patch/driver-support-optoe-twoaddr-a2h-access.patch

```diff
@@ -0,0 +1,69 @@
+From 23a58d78c8c12b0092e1dfded96e6d648bffe1de Mon Sep 17 00:00:00 2001
+From: Jemston Fernando <jemston_fernando@dell.com>
+Date: Thu, 25 Nov 2021 04:30:04 -0800
+Subject: [PATCH] A2h access for optoe2 even when DDM is unsupported
+ Enable 0x51 (A2h) access for optoe2 even when DDM is unsupported, for
+ accessing vendor specific fields. Some SFP optics that do not support DDM
+ can have vendor specific implementation via 0x51 in A2h register space
+ (eg. Mailbox Registers).
+
+ Removed the DDM support check for optoe2 (TWO_ADDR) and now its EEPROM
+ size is 512 bytes minimum. Return error when accessing beyond 256 bytes
+ for optics that do not support I2C address 0x51.
```

Comment:
Please add a blank line between the commit message summary and body. Please do not indent the body by a space. (Please read [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/).)

Reply:
I've implemented your comments. Thanks for the review.

P.S. The link (How to Write a Git Commit Message) you pasted earlier had a different hyperlink and it didn't work.